app/views/external_users/claims/supporting_documents/_new.html.haml

```diff
@@ -1,5 +1,5 @@
 - unless @claim.agfs? && @claim.interim?
-  %h2.govuk-heading-l
+  %h2.govuk-heading-l{ class: 'govuk-!-static-margin-bottom-0'}
```

Comment:
Just checking that `0` is the correct margin here? (Not suggesting you need to change it, just a sense-check that it is what is actually intended 🙂)